phalcon/Mvc/Application.zep

```diff
@@ -79,13 +79,17 @@ class Application extends AbstractApplication
     /**
      * Handles a MVC request
      */
-    public function handle(string! uri) -> <ResponseInterface> | bool
+    public function handle(var! uri)
```

Comment:
But why? Previously users was able to use this like:
```php
    $application = new Application($di);

    echo $application->handle()
        ->getContent();
```

What approach do you suggest? It is breaks BC (again). Anyway don't forget to update docs, as well as change log please.

Reply:
And we can leave this the way it was.